source/agora/consensus/state/ValidatorSet.d

```diff
@@ -571,6 +593,50 @@ public class ValidatorSet
 
         return true;
     }
+
+    /***************************************************************************
+
+        Find the most recent Enrollment with the provided UTXO hash, regardless
+        of it's active status
+
+        Params:
+            enroll_key = The key for the enrollment
+            state = struct to fill once an enrollment is found
+
+        Returns:
+            true if an enrollment with the key was ever accepted, false otherwise
+
+    ***************************************************************************/
+
+    bool findRecentEnrollment (in Hash enroll_key, out EnrollmentState state) @trusted nothrow
```

Comment:
Could use a test-case in the module.

Reply:
Added the test for it in the `EnrollmentManager.d`